hardware/memsys_test_mem.mk

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2019, University of Washington All rights reserved.
```

Comment:
Since these .mk files are breeding we should put them somewhere

Reply:
You mean we should take the old ones out to the farm?

Reply:
Uh, sure. 

But what I really meant was to put them all in a folder. Least of my concerns.